### PR TITLE
Out of gas  in return/revert

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/GeneralStateTestsFiller/stMemExpandingEIP150Calls/OOGinReturnFiller.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+          ; Other constants
+          (def 'NOP 0)   ; No OPeration
+
+          ; Understand the input.
+          [type]       $4
+          [gas2Use]    $36
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+          [0] 0x60A760A7
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stMemExpandingEIP150Calls/OOGinReturnFiller.yml:60]: what if there is no memory set before the call? you mentioned there was some strange buffer

**Review by winsvega — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -27,14 +27,13 @@ sani-vm: $(vm_tests:=.format) $(vm_fillers:=.format) \
          $(vm_tests:=.filled)
 
 # TODO: enable $(gs_fillers:=.valid) $(gs_tests:=.format) $(gs_fillers:=.format)
-sani-gs: $(gs_tests:=.valid) \
```
Comment (winsvega) [inline Makefile:30]: can you make it a separate pr and commit as a final version of this file to develop

**Comment by qbzzt** [comment]
Replaced by PR 743
